lametro/static/css/wagtail_custom.css

```diff
@@ -0,0 +1,53 @@
+[class*="alert"] {
```

Comment:
Low key obsessed with CSS variables.

Reply:
Did some reading up on this and oh yea I can totally see why 👀